sdk/authorization/pom.xml

```diff
@@ -8,7 +8,4 @@
   <artifactId>azure-authorization-service</artifactId>
```

Comment:
Do we want to keep this pom.xml and ci.yml in this sdk/authorization directory? There are no other artifacts here.